{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "5ebd4abd-af55-4e11-911c-a98e101054ac",
   "metadata": {},
   "source": [
    "This notebook has examples of how to use the CDS API to do common queries and return results that should be usable, and exportable to a Cloud Resource. A few useful links:\n",
    "\n",
    "    The CDS Data Model: https://dataservice.datacommons.cancer.gov/#/resources\n",
    "    The CDS GraphQL endpoint: https://dataservice.datacommons.cancer.gov/v1/graphql/\n",
    "    The GraphiQL interface in CDS (a good place to build queries): https://dataservice.datacommons.cancer.gov/#/graphql\n",
    "\n",
    "Import a few useful libraries. Requests for carrying out the communications and Pandas for manipulating the returns. We also need json to help with formatting the results.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "03edc027-8dec-47da-98f4-7d8879463e91",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import requests"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "b0f1c5f4-72b3-4496-9e4a-5990dd2b49b7",
   "metadata": {},
   "outputs": [],
   "source": [
    "cds_graphql_url = \"https://dataservice.datacommons.cancer.gov/v1/graphql/\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b7a98a6e-5527-46a4-ac58-dfbf28c91769",
   "metadata": {},
   "source": [
    "Since there will be multiple queries, having a simple routine that runs the query and returns the answer as a JSON object will streamline the notebook"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "eea98644-eaba-4edf-99fa-87abe4d9fa57",
   "metadata": {},
   "outputs": [],
   "source": [
    "def runGraphQLQuery(url,query):\n",
    "    headers = {\"content-type\":\"application/json\"}\n",
    "    try:\n",
    "        results = requests.post(url = url, json={\"query\":query, \"headers\":headers})\n",
    "        results = results.json()\n",
    "    except requests.exceptions.HTTPError as exception:\n",
    "        print(exception)\n",
    "    return results"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "33a355d6-6c42-4ecf-9565-657899d9eef6",
   "metadata": {},
   "source": [
    "CDS has multiple studies, so as a first step, it's worth taking a look to see what content is avaialble.  This first query simply lists out the different studies that have data (This notebook assumes that you've done some basic introspection queries to get familiar wtih the content of the CDS database)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "49225387-eec8-4630-a51a-b9844b2e787c",
   "metadata": {},
   "outputs": [],
   "source": [
    "study_query = \"\"\"\n",
    "{\n",
    "  study{\n",
    "    study_name\n",
    "    study_description\n",
    "    study_acronym\n",
    "    study_access\n",
    "    phs_accession\n",
    "  }\n",
    "}\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ee9dfb3b-5b5b-4f2f-b0c4-8afa83718774",
   "metadata": {},
   "source": [
    "This query will bring back several useful pieces of information that should help decide what studies are worth further examination.  You can also run this query in the GraphiQL interface in the CDS portal (https://dataservice.datacommons.cancer.gov/#/graphql) to see what the JSON object looks like."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "74596d61-53b5-4d3c-8c35-cca04f4f6852",
   "metadata": {},
   "outputs": [],
   "source": [
    "study_result = runGraphQLQuery(cds_graphql_url, study_query)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "132a795a-e368-4efb-9e5c-d43af77f0e30",
   "metadata": {},
   "source": [
    "Let's put the information in a dataframe so we can easily make it more readable than a JSON object"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "9032f4bf-427a-4987-8d67-d039f28dd55f",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_columns = ['Study Name', 'Acronym', 'Access', 'Accession', 'Description']\n",
    "study_df = pd.DataFrame(columns = df_columns)\n",
    "for study in study_result['data']['study']:\n",
    "    study_df.loc[len(study_df.index)] = [study['study_name'],study['study_acronym'],study['study_access'], study['phs_accession'],study['study_description']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "fba04fe3-4b5e-4f95-a143-26cd7c1cbd6e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Study Name</th>\n",
       "      <th>Acronym</th>\n",
       "      <th>Access</th>\n",
       "      <th>Accession</th>\n",
       "      <th>Description</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Clinical Trial Specimen Molecular Characterization (CTSMC)</td>\n",
       "      <td>CTSMC</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002790</td>\n",
       "      <td>The National Cancer Institute's (NCI) Childhood Cancer Data Initiative (CCDI) focuses on the critical need to collect, analyze, and share data to address the burden of cancer in children, adolescents, and young adults (AYAs). The Molecular Characterization Initiative (MCI) will further the CCDI's goals by providing access to better diagnostic tests for pediatric and AYA patients. The molecular characterizations of solid tumors, soft tissue sarcomas, and rare diseases are performed in a CLIA-certified setting as results may be used to screen for and/or confirm clinical trial eligibility, direct treatment, or otherwise contribute to the conduct of the trial. The following molecular characterization assays were performed:\\n\\nTumor/Normal Whole Exome sequencing\\nMethylation arrays\\nArcher fusion panel\\nDeidentified clinical reports\\nNote: New data will be added periodically, please check the SBG site for updates.\\n\\nStudy Weblinks:\\nCCDI Molecular Characterization Initiative\\nStudy Design:\\nTumor vs. Matched-Normal\\nStudy Type:\\nAggregate Genomic Data\\nClinical Diagnostic Testing\\nExome Sequencing\\nIndividual-Level Genomic Data\\nTranscriptome Analysis\\nTumor vs. Matched-Normal\\nTotal number of consented subjects: 1996</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>UCSF Database for the Advancement of JMML - Integration of Metadata with \"\"Omic\"\" Data</td>\n",
       "      <td>JMML</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002504</td>\n",
       "      <td>Juvenile myelomonocytic leukemia (JMML) is a rare and frequently fatal myeloproliferative/myelodysplastic disorder of early childhood with an estimated incidence of 1.2 cases per million. It is associated with a spectrum of diverse outcomes ranging from spontaneous resolution in rare patients to transformation to acute myeloid leukemia in others. The overwhelming majority of JMML patients (~95%) will harbor mutations in canonical Ras pathway genes, including NF1, NRAS, KRAS, PTPN11, and CBL. As Ras proteins are mutated in more than 30% of human cancers, the information gleaned from the study of JMML has provided insights into Ras signaling in cancer as well as a group of congenital diseases with tumor predispositions known as the “Rasopathies”. While oncogenic Ras is one of the most common mutations in human cancer, it remains one of the most vexing targets for efficacious therapy.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Human Tumor Atlas Network (HTAN) primary sequencing data</td>\n",
       "      <td>HTAN</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002371</td>\n",
       "      <td>An NCI-funded Cancer Moonshot initiative to construct 3-dimensional atlases of the dynamic cellular, morphological, and molecular features of human cancers as they evolve from precancerous lesions to advanced disease.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>CIDR: Discovery, Biology, and Risk of Inherited Variants in Glioma sample</td>\n",
       "      <td>None</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002250</td>\n",
       "      <td>This is a gliogene brain tumor family study. This study includes glioma cases with a family history of glioma in a first, second, or third degree relative. \\nStudy Weblinks:\\nGliogene\\nStudy Design:\\nFamily/Twin/Trios\\nStudy Type:\\nFamily\\nWhole Genome Sequencing\\ndbGaP estimated ancestry using GRAF-pop\\nTotal number of consented subjects: 151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Childhood Cancer Data Initiative (CCDI): Free the Data: Open Sharing of Comprehensive Genomic Childhood Cancer Datasets (Kansas)</td>\n",
       "      <td>CCDI-KUMC</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002529</td>\n",
       "      <td>This study provides paired tumor normal genomic sequencing data from approximately 200 children with cancer, including both solid tumors and leukemias, done by the Children’s Mercy Research Institute (CMRI) and University of Kansas Cancer Center (KUCC).\\n\\nThese data include whole genome sequencing (generally, ~20x), whole exome sequencing (generally, ~300x), bulk RNA sequencing (generally, ~80 million reads), and single-cell RNA and ATAC sequencing (&gt;50,000 reads/cell). Additional phenotypic, pathologic, and genetic data, gathered clinically for these samples, are also provided.\\nStudy Design:\\n    Tumor vs. Matched-Normal\\nStudy Type:\\nCase Set\\nClinical Cohort\\nClinical Genetic Testing\\nExome Sequencing\\nFull Transcriptome Sequencing\\nIndividual-Level Genomic Data\\nMixed\\nProbands\\nRepository\\nRNA Sequencing\\nSequencing\\nSingle Cell Analysis\\nTranscriptome Sequencing\\nTumorTumor vs. Matched-Normal\\nWhole Genome Sequencing\\nTotal number of consented subjects: 193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>University of Texas PDX Development and Trial Center Grant</td>\n",
       "      <td>None</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs001980</td>\n",
       "      <td>The goal for the University Texas PDX Development and Trial Center (UTPDTC) is to optimize personalized biomarker-based cancer therapy and identify effective targeted drugs based on the molecular characteristics of each tumor. Our short-term goals are to establish a biobank of clinically, and molecularly-annotated Patient-Derived Xenografts (PDXs) and to use PDXs as a platform for preclinical drug development and biomarker discovery. The primary goal for UTPDTC investigators will be to develop PDX trial strategies for preclinical testing of single agents and drug combinations. These models will allow the determination of the optimal treatments (single drugs or combinations) that should be tested in clinical trials in increasingly individualized, molecularly defined subsets of tumors. The goal of the Patient-Derived Xenograft Core is to provide high-quality clinically relevant and molecularly annotated PDX models for the research projects proposed in the University of Texas PDX Development and Trial Center (UTPDTC) grant application and to the research activity of the NCI PDX Development and Trial Centers Research Network (PDXNet) by leveraging PDX resources at our institutions and developing new PDX models from human cancer specimens using rigorous quality standards so that the models can be used to guide clinical trial development. The PDX models developed and/or characterized by the PDX Core will be available to other cancer researchers through the PDXNet and NCI Patient-Derived Models Repository (PDMR).\\n\\n    Study Design:\\n        Prospective Longitudinal Cohort\\n    Study Type:\\n        Cohort\\n    Total number of consented subjects: 36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Clonal evolution during metastatic spread in high-rish neuroblastoma</td>\n",
       "      <td>CEMSHRN</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs003111</td>\n",
       "      <td>The goal of this study is to deliver a detailed characterization of the patterns of disease dissemination at diagnosis, during progression and in response to therapy in high-risk neuroblastoma. Longitudinal and spatially distinct tumors were collected from patients. Clinical data includes diagnosis and treatment information. Biospecimen data includes whole genome sequencing (WGS) and whole transcriptome sequencing (WTS).\\n\\nStudy Design:\\nTumor vs. Matched-Normal\\nStudy Type:\\nLongitudinal Cohort\\nTranscriptome Sequencing\\nWhole Genome Sequencing\\nTotal number of consented subjects: 129</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>LCCC 1108: Development of a Tumor Molecular Analyses Program and Its Use to Support Treatment Decisions (UNCseqTM)</td>\n",
       "      <td>LCCC 1108 (UNCseqTM)</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs001713</td>\n",
       "      <td>The primary objective of this specimen correlative study was two-fold: to provide a mechanism for the association of known molecular alterations with clinical outcomes, and to provide rapid genetic profiling of alterations with known clinical utility using tumor and germline specimens to support treatment decisions.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Washington University PDX Development and Trial Center</td>\n",
       "      <td>None</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002305</td>\n",
       "      <td>This data set is a collection of patient specimens from common and rare cancers that have been developed into patient-derived xenograft (PDX) models at the Washington University PDX Development and Trial Center (WU-PDTC). The goal of the PDTC is to develop and characterize PDX models, gaining insight into tumor biology, and validating biomarkers across all major tumor types. Pre-clinical experiments in these PDX models will be used to advance our ability to predict clinical responses to new molecularly targeted agents under development.\\n\\nStudy Design:\\nTumor vs. Matched-Normal\\nStudy Type:\\nTumor\\nTumor vs. Matched-Normal\\nTotal number of consented subjects: 127</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>CPTAC Proteogenomic Study</td>\n",
       "      <td>PanCanSnATAC</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs001287</td>\n",
       "      <td>Recently, significant progress has been made in characterizing and sequencing the genomic alterations in statistically robust numbers of samples from several types of cancer. For example, The Cancer Genome Atlas (TCGA), International Cancer Genome Consortium (ICGC) and other similar efforts are identifying genomic alterations associated with specific cancers (e.g., copy number aberrations, rearrangements, point mutations, epigenomic changes, etc.) The availability of these multi-dimensional data to the scientific community sets the stage for the development of new molecularly targeted cancer interventions. Understanding the comprehensive functional changes in cancer proteomes arising from genomic alterations and other factors is the next logical step in the development of high-value candidate protein biomarkers. Hence, proteomics can greatly advance the understanding of molecular mechanisms of disease pathology via the analysis of changes in protein expression, their modifications and variations, as well as protein=protein interaction, signaling pathways and networks responsible for cellular functions such as apoptosis and oncogenesis. Realizing this great potential, the NCI launched the third phase of the CPTC initiative in September 2016. As the Clinical Proteomic Tumor Analysis Consortium, CPTAC continues to define cancer proteomes on genomically-characterized biospecimens. The purpose of this integrative approach was to provide the broad scientific community with knowledge that links genotype to proteotype and ultimately phenotype. In this third phase of CPTAC, the program aims to expand on CPTAC II and genomically and proteomically characterize over 2000 samples from 10 cancer types (Lung Adenocarcinoma, Pancreatic Ductal Adenocarcinoma, Glioblastoma Multiforme, Acute Myeloid Leukemia, Clear cell renal Carcinoma, Head and Neck Squamous Cell Carcinoma, Cutaneous Melanoma, Sarcoma, Lung Squamous Cell Carcinoma, Uterine Corpus Endometrial Carcinoma) .Germline DNA is obtained from blood and Normal control samples for proteomics varied by organ site. All cancer samples were derived from primary and untreated tumor.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NCI CCSG CCDI Supplement Additional Genomic Submission</td>\n",
       "      <td>None</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002599</td>\n",
       "      <td>The implementation of targeted therapies for acute myeloid leukaemia (AML) has been challenging because of the complex mutational patterns within and across patients as well as a dearth of pharmacologic agents for most mutational events. This supplement data from the Beat AML program provides Adolescent and Young Adult (AYA) and pediatric tumor specimens not previously released. These data include exome and RNA level characterization as well as analyses of ex vivo drug sensitivity. Collectively, this data can be leveraged to address clinical, genomic, transcriptomic and functional analyses of the biology of leukemia in pediatric and AYA populations.\\nStudy Design:\\nCase Set\\nStudy Type:\\nCase Set\\nExome Sequencing\\nRNA Sequencing\\nTotal number of consented subjects: 104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>CIDR: The Genetic Basis of Aggressive Prostate Cancer: The Role of Rare Variation</td>\n",
       "      <td>None</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs001524</td>\n",
       "      <td>In this study, we will apply a multi-staged approach to reveal genes harboring rare variants that are associated with aggressive PCa. Whole-exome sequencing (Aim 1a) of 2,774 aggressive cases and 2,776 non-aggressive cases of European ancestry will be conducted followed by rare variant analysis of single sites and gene burden testing to identify novel susceptibility loci/genes for aggressive disease. We will validate the most significantly associated genes (~500) through targeted sequencing in an additional 7,500 aggressive and 7,500 non-aggressive cases (Aim 1b). Next, we will investigate the clinical predictive utility of the genes/variants identified in 2,300 cases in the STHM3 trial who are undergoing biopsy based on PSA and genetic risk score stratification (Aim 2). Through this tiered approach we expect to significantly advance knowledge of aggressive PCa etiology and health disparities as well as guide the development of early detection and prognostic strategies for the subset of men who are most susceptible to this fatal form of disease. In this case-case study of aggressive vs non aggressive prostate cancer, aggressive cases are defined as prostate cancer as cause of death, (T4 disease or T3 disease) and Gleason 8+. Non-aggressive cases are men with T1/2 disease and Gleason &lt;=6.\\n\\nCONTRIBUTING SITES\\n\\nCAPS, PROCAP, STHM1, STHM2: Swedish Cancer Society (CAN 2016/818), Swedish Research Council (2014/2269)\\n\\nMEC: Funding provided by the National Cancer Institute: Understanding Ethnic Differences in Cancer, 2U01CA164973 and The Genetic Basis of Aggressive Prostate Cancer, The Role of Rare Variation, 5R01CA196931-02.\\n\\nCPSII: The authors express sincere appreciation to all Cancer Prevention Study II participants and to each member of the study and biospecimen management group. The American Cancer Society funds the creation, maintenance, and updating of the Cancer Prevention Study-II cohort.\\n\\nMCCS/APCS/PCFS: The Melbourne Collaborative Cohort Study (MCCS) recruitment was funded by VicHealth and Cancer Council Victoria and further supported by Australian National Health and Medical Research Council (NHMRC) grants 209057 and 396414. The Aggressive Prostate Cancer Case-Control Study (APCS) was funded by NHMRC grant 623204. The Prostate Cancer Family Study (PCFS) was fully funded by Cancer Council Victoria. Cancer Council Victoria funds the continuing maintenance and updating of the MCCS, APCS and PCFS. Cases and their vital status are ascertained and followed up through the Victorian Cancer Registry and the Australian Institute of Health and Welfare, including the National Death Index and the Australian Cancer Database.\\n\\nPLCO: The Prostate Lung Colorectal Ovarian Cancer Screening Trial (PLCO) was supported by the Intramural Research Program of the Division of Cancer Epidemiology and Genetics and by contracts from the Division of Cancer Prevention, National Cancer Institute, US National Institutes of Health, Department of Health and Human Services.\\n\\nATBC: The ATBC Study is supported by the Intramural Research Program of the U.S. National Cancer Institute, National Institutes of Health, and by U.S. Public Health Service contract HHSN261201500005C from the National Cancer Institute, Department of Health and Human Services.\\n\\nICR: This work was supported by the NIH R01 grant 5R01CA196931-02.The samples from the UK were from UKGPCS and PrompT. The UKGPCS study was supported by Cancer Research UK (grant numbers C5047/A7357, C1287/A10118, C1287/A5260, C5047/A3354, C5047/A10692, C16913/A6135 and C16913/A6835). We would like to acknowledge the NCRN nurses and Consultants for their work in the UKGPCS study. We thank all the patients who took part in this study. We also acknowledge The Institute of Cancer Research, The National Cancer Research Network UK, The National Cancer Research Institute (NCRI) UK for their ongoing support. We are grateful for support of NIHR funding to the NIHR Biomedical Research Centre at The Institute of Cancer Research and The Royal Marsden NHS Foundation Trust.\\n\\nCIDR grant X01HG008336\\n\\nStudy Design:\\nCase Set\\nStudy Type:\\nCase Set\\ndbGaP estimated ancestry using GRAF-pop\\nTotal number of consented subjects: 5563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Feasibility and Clinical Utility of Whole Genome Profiling in Pediatric and Young Adult Cancers</td>\n",
       "      <td>None</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002620</td>\n",
       "      <td>Comprehensive Whole Genome Sequencing and Whole Transcriptome (RNASeq) analyses from 114 pediatric patients at Memorial Sloan Kettering. This dataset was used as a prospective clinical utility and feasibility study. By using our Isabl infrastructure for precision medicine, we developed a 2-week end-to-end pipeline to analyze cfDNA, WGS and WTS, which we refer to as cWGTS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>CIDR: The Role of Rare Coding Variation in Prostate Cancer in Men of African Ancestry - RESPOND Project 2</td>\n",
       "      <td>None</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002637</td>\n",
       "      <td>\"In RESPOND Project 2, we seek to identify rare genetic factors that are associated with prostate cancer (PCa) risk and aggressiveness in men of African ancestry (AA). We will conduct exome sequencing of 15,000 prostate cancer cases and 5,000 controls from the RESPOND cohort and the African Ancestry Prostate Cancer Consortium (AAPC) with cases selected based on risk categories: high-risk (stage T3/T4 or Gleason 8+ or PSA&gt;20 ng/ml), intermediate-risk (stage T2b/T2c or Gleason 7 or PSA 10-20 ng/ml) and low-risk disease (stage T1/T2a and Gleason ≤ 6 and PSA&lt;10 ng/ml). We expect the findings from this Project to significantly advance knowledge of susceptibility to aggressive PCa and racial/ethnic disparities in PCa risk, and to guide the development of future preventive, early detection and prognostic measures for AA men. The first phase of the study will include exome sequence data for approximately 7,500 cases and 5,000 controls from AAPC. The second phase of the study will include exome data for 5,000 cases from RESPOND and AAPC.\\n\\nACKNOWLEDGMENTS and CONTRIBUTING SITES\\n\\nMultiethnic Cohort (MEC): The MEC and the genotyping in this study were supported by National Institutes of Health (NIH) grants CA63464, CA54281, CA1326792, CA148085, and HG004726. Cancer incidence data for the MEC and Los Angeles Study of Aggressive Prostate Cancer (LAAPC) studies have been collected by the Los Angeles Cancer Surveillance Program of the University of Southern California with federal funds from the National Cancer Institute (NCI)/NIH/Department of Health and Human Services (DHHS) under Contract No. N01-PC-35139, and the California Department of Health Services as part of the state-wide cancer reporting program mandated by California Health and Safety Code Section 103885, and grant 1U58DP000807-3 from the Centers for Disease Control and Prevention (CDC).\\n\\nGhana Prostate Study (GPS): The Ghana Prostate Study was funded by the Intramural Program of the NCI/NIH/DHHS under Contract No. HHSN261200800001E.\\n\\nMen of African Descent and Carcinoma of the Prostate (MADCaP): We thank all MADCaP study participants. This work is a product of the MADCaP network. This work was supported by NCI/NIH grant U01CA184374 to Timothy Rebbeck and National Institute of General Medical Sciences (NIGMS) MIRA grant R35GM133727 to Joseph Lachance. Additional funding includes a seed grant from the Integrated Cancer Research Center at Georgia Institute of Technology.\\n\\nUGANDA: The UGANDA study was supported by NIH grant R01CA165862.\\n\\nSouthern Community Cohort (SCCS) is funded by NIH grant CA092447. SCCS sample preparation was conducted at the Epidemiology Biospecimen Core Lab that is supported in part by the Vanderbilt Ingram Cancer Center (CA68485). Data on SCCS cancer cases were provided by Alabama Statewide Cancer Registry, Kentucky Cancer Registry, Office of Cancer Surveillance at Tennessee Department of Health, Florida Cancer Data System, Central Cancer Registry at North Carolina Division of Public Health, Georgia Comprehensive Cancer Registry, Louisiana Tumor Registry, Mississippi Cancer Registry, South Carolina Central Cancer Registry, Virginia Cancer Registry at Virginia Department of Health, and Cancer Registry at Arkansas Department of Health. The Arkansas Central Cancer Registry is fully funded by a grant from National Program of Cancer Registries (NPCR)/Centers for Disease Control and Prevention (CDC). Data on SCCS cancer cases from Mississippi were collected by the Mississippi Cancer Registry which participates in the NPCR/CDC. The contents of this publication are solely the responsibility of the authors and do not necessarily represent the official views of the CDC or the Mississippi Cancer Registry.\\n\\nKaruprostate: The Karuprostate study was supported by the French National Health Directorate and by the Association pour la Recherche sur les Tumeurs de la Prostate. Séverine Ferdinand, Marc Romana.\\n\\nThe North Carolina - Louisiana Prostate Cancer Project (PCaP) is carried out as a collaborative study supported by the Department of Defense contract DAMD 17-03-2-0052. The authors thank the staff, advisory committees and research subjects participating in the PCaP study for their important contributions. We would also like to acknowledge the UNC BioSpecimen Facility and the LSUHSC Pathology Lab for our DNA extractions, blood processing, storage and sample disbursement (https://genome.unc.edu/bsp).\\n\\nGene-Environment Interaction in Prostate Study (GECAP) was supported by NIH grant ES011126.\\n\\nKing County Prostate Cancer Study (KCPCS) was supported by NIH grants CA056678, CA082664, and CA092579, with additional support from the Fred Hutchinson Cancer Research Center. We thank the participants in these studies, and Ms. Suzanne Kolb for help with study management.\\n\\nThe Los Angeles Study of Aggressive Prostate Cancer (LAAPC) was funded by grant 99-00524V-10258 from the Cancer Research Fund, under Interagency Agreement #97-12013 (University of California contract #98-00924V) with the Department of Health Services Cancer Research Program. Cancer incidence data for the MEC and LAAPC studies have been collected by the Los Angeles Cancer Surveillance Program of the University of Southern California with Federal funds from the NCI/NIH/DHHS under Contract No. N01-PC-35139, and the California Department of Health Services as part of the state-wide cancer reporting program mandated by California Health and Safety Code Section 103885, and grant 1U58DP000807-3 from the CDC.\\n\\nProstate Cancer Studies at MD Anderson (MDA) was supported by grants CA68578, ES007784, DAMD W81XWH-07-1-0645, and CA140388.\\nStudy Design:\\nCase-Control\\nStudy Type:\\nCase-Control\\nCohort\\nExome Sequencing\\nTotal number of consented subjects: 12049\"</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Childhood Cancer Data Initiative (CCDI): Integration of genomic and clinical data from unique rare cancer datasets to facilitate data sharing</td>\n",
       "      <td>CBTN</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002517</td>\n",
       "      <td>This sequencing cohort defines the largest, clinically annotated pediatric brain tumor cohort study to date and seeks to define the intersection of germline and somatic underpinnings of pediatric brain tumors across a shared developmental context of cancer and structural birth defects.\\nStudy Weblinks:\\nChildrens Brain Tumor Network\\nPacific Pediatric Neuro-Oncology Consortium\\nCHOP Division of Genomic Diagnostics\\nKids First Data Resource Portal\\nPedCBio Portal\\nStudy Design:\\nTumor vs. Matched-Normal\\nStudy Type:\\nClinical Cohort\\nClinical Genetic Testing\\nClinical Trial\\nCopy Number Variation (CNV)\\nExome Sequencing\\nGenotype/Expression\\nIndividual-Level Genomic Data\\nLongitudinal\\nLongitudinal Cohort\\nMulticenter\\nObservational\\nParent-Offspring\\nParent-Offspring Trios\\nProbands\\nProspective\\nRNA Sequencing\\nSequencing\\nTissue Expression\\nTumor\\nTumor vs. Matched-Normal\\nWhole Genome Sequencing\\nTotal number of consented subjects: 1031</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Wistar PDX Development and Trial Center SPORE in Skin Cancer</td>\n",
       "      <td>None</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs002432</td>\n",
       "      <td>Overall the outcomes of patients with metastatic melanoma have improved dramatically over the last decade due to an improved understanding of the molecular drivers of this disease. In particular, multiple targeted therapy regimens have been approved for patients with a BRAFV600E/K mutation, which are present in ~50% of cutaneous melanomas. These treatments achieve clinical responses in ~80% of patients with a BRAFV600E/K mutation, thus providing proof-of-concept of the therapeutic potential for personalized therapeutic strategies. However, most of the patients will progress within 2 years of starting those therapies. Further, currently there are no targeted therapies that have been shown to be effective in patients with a wild-type BRAF. Thus, there are unmet clinical needs to develop treatments that prevent or overcome resistance to existing therapies for patients with a BRAFV600E/K mutation, and that are effective in patients without a BRAF mutation. In order to facilitate the development of new therapeutic strategies, over the last 5 years we have led a major effort to develop a broad collection of PDX models to reflect the clinical, histological, and genetic heterogeneity of this disease. Our collection of PDX models represents one of the largest collections for any human malignancy, and our initial testing demonstrates that the collection accurately recapitulates the oncogenic drivers and molecular heterogeneity that is observed in patients. This collection also includes a subset of PDX established from patients with acquired resistance to targeted therapies that have been maintained on those agents in vivo to sustain their resistant phenotype. Together these efforts have generated a robust resource to develop, refine, and prioritize new personalized combinatorial therapies for patients. Thus, we propose to establish a multi-disciplinary and multi-institutional PDTC Program focused on the use and continued expansion of our robust melanoma PDX collection to identify new therapeutic approaches that fill important clinical gaps in this disease.\\nStudy Design:\\nCase Set\\nStudy Type:\\nCase Set\\nExome Sequencing\\nRNA Sequencing\\nTotal number of consented subjects: 46</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>TCGA WGS Variants Across 18 Cancer Types</td>\n",
       "      <td>None</td>\n",
       "      <td>Controlled</td>\n",
       "      <td>phs003155</td>\n",
       "      <td>Even though whole genome sequence (WGS) data has been generated and published in many studies, much of this information is not yet processed for use in down-stream analysis. This study's aim is to provide 1342 WGS normal-tumor paired single nucleotide variations (SNP) over 18 different cancer types provided by The Cancer Genome Atlas (TCGA) project. Individual level data for TCGA can be accessed by requesting access for phs000178. This data includes variations within self-reported white and African-American populations. Variations that exist within the tumor tissue but are absent in associated normal organ tissue (as compared to the human reference genome) are reported. Data published includes SNP and small insertions and deletions, which were generated through a pipeline including the VarScan2 variant calling software.</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                       Study Name  \\\n",
       "0                                                                                      Clinical Trial Specimen Molecular Characterization (CTSMC)   \n",
       "1                                                          UCSF Database for the Advancement of JMML - Integration of Metadata with \"\"Omic\"\" Data   \n",
       "2                                                                                        Human Tumor Atlas Network (HTAN) primary sequencing data   \n",
       "3                                                                       CIDR: Discovery, Biology, and Risk of Inherited Variants in Glioma sample   \n",
       "4                Childhood Cancer Data Initiative (CCDI): Free the Data: Open Sharing of Comprehensive Genomic Childhood Cancer Datasets (Kansas)   \n",
       "5                                                                                      University of Texas PDX Development and Trial Center Grant   \n",
       "6                                                                            Clonal evolution during metastatic spread in high-rish neuroblastoma   \n",
       "7                              LCCC 1108: Development of a Tumor Molecular Analyses Program and Its Use to Support Treatment Decisions (UNCseqTM)   \n",
       "8                                                                                          Washington University PDX Development and Trial Center   \n",
       "9                                                                                                                       CPTAC Proteogenomic Study   \n",
       "10                                                                                         NCI CCSG CCDI Supplement Additional Genomic Submission   \n",
       "11                                                              CIDR: The Genetic Basis of Aggressive Prostate Cancer: The Role of Rare Variation   \n",
       "12                                                Feasibility and Clinical Utility of Whole Genome Profiling in Pediatric and Young Adult Cancers   \n",
       "13                                      CIDR: The Role of Rare Coding Variation in Prostate Cancer in Men of African Ancestry - RESPOND Project 2   \n",
       "14  Childhood Cancer Data Initiative (CCDI): Integration of genomic and clinical data from unique rare cancer datasets to facilitate data sharing   \n",
       "15                                                                                   Wistar PDX Development and Trial Center SPORE in Skin Cancer   \n",
       "16                                                                                                       TCGA WGS Variants Across 18 Cancer Types   \n",
       "\n",
       "                 Acronym      Access  Accession  \\\n",
       "0                  CTSMC  Controlled  phs002790   \n",
       "1                   JMML  Controlled  phs002504   \n",
       "2                   HTAN  Controlled  phs002371   \n",
       "3                   None  Controlled  phs002250   \n",
       "4              CCDI-KUMC  Controlled  phs002529   \n",
       "5                   None  Controlled  phs001980   \n",
       "6                CEMSHRN  Controlled  phs003111   \n",
       "7   LCCC 1108 (UNCseqTM)  Controlled  phs001713   \n",
       "8                   None  Controlled  phs002305   \n",
       "9           PanCanSnATAC  Controlled  phs001287   \n",
       "10                  None  Controlled  phs002599   \n",
       "11                  None  Controlled  phs001524   \n",
       "12                  None  Controlled  phs002620   \n",
       "13                  None  Controlled  phs002637   \n",
       "14                  CBTN  Controlled  phs002517   \n",
       "15                  None  Controlled  phs002432   \n",
       "16                  None  Controlled  phs003155   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Description  \n",
       "0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 The National Cancer Institute's (NCI) Childhood Cancer Data Initiative (CCDI) focuses on the critical need to collect, analyze, and share data to address the burden of cancer in children, adolescents, and young adults (AYAs). The Molecular Characterization Initiative (MCI) will further the CCDI's goals by providing access to better diagnostic tests for pediatric and AYA patients. The molecular characterizations of solid tumors, soft tissue sarcomas, and rare diseases are performed in a CLIA-certified setting as results may be used to screen for and/or confirm clinical trial eligibility, direct treatment, or otherwise contribute to the conduct of the trial. The following molecular characterization assays were performed:\\n\\nTumor/Normal Whole Exome sequencing\\nMethylation arrays\\nArcher fusion panel\\nDeidentified clinical reports\\nNote: New data will be added periodically, please check the SBG site for updates.\\n\\nStudy Weblinks:\\nCCDI Molecular Characterization Initiative\\nStudy Design:\\nTumor vs. Matched-Normal\\nStudy Type:\\nAggregate Genomic Data\\nClinical Diagnostic Testing\\nExome Sequencing\\nIndividual-Level Genomic Data\\nTranscriptome Analysis\\nTumor vs. Matched-Normal\\nTotal number of consented subjects: 1996  \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Juvenile myelomonocytic leukemia (JMML) is a rare and frequently fatal myeloproliferative/myelodysplastic disorder of early childhood with an estimated incidence of 1.2 cases per million. It is associated with a spectrum of diverse outcomes ranging from spontaneous resolution in rare patients to transformation to acute myeloid leukemia in others. The overwhelming majority of JMML patients (~95%) will harbor mutations in canonical Ras pathway genes, including NF1, NRAS, KRAS, PTPN11, and CBL. As Ras proteins are mutated in more than 30% of human cancers, the information gleaned from the study of JMML has provided insights into Ras signaling in cancer as well as a group of congenital diseases with tumor predispositions known as the “Rasopathies”. While oncogenic Ras is one of the most common mutations in human cancer, it remains one of the most vexing targets for efficacious therapy.  \n",
       "2                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         An NCI-funded Cancer Moonshot initiative to construct 3-dimensional atlases of the dynamic cellular, morphological, and molecular features of human cancers as they evolve from precancerous lesions to advanced disease.  \n",
       "3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         This is a gliogene brain tumor family study. This study includes glioma cases with a family history of glioma in a first, second, or third degree relative. \\nStudy Weblinks:\\nGliogene\\nStudy Design:\\nFamily/Twin/Trios\\nStudy Type:\\nFamily\\nWhole Genome Sequencing\\ndbGaP estimated ancestry using GRAF-pop\\nTotal number of consented subjects: 151  \n",
       "4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               This study provides paired tumor normal genomic sequencing data from approximately 200 children with cancer, including both solid tumors and leukemias, done by the Children’s Mercy Research Institute (CMRI) and University of Kansas Cancer Center (KUCC).\\n\\nThese data include whole genome sequencing (generally, ~20x), whole exome sequencing (generally, ~300x), bulk RNA sequencing (generally, ~80 million reads), and single-cell RNA and ATAC sequencing (>50,000 reads/cell). Additional phenotypic, pathologic, and genetic data, gathered clinically for these samples, are also provided.\\nStudy Design:\\n    Tumor vs. Matched-Normal\\nStudy Type:\\nCase Set\\nClinical Cohort\\nClinical Genetic Testing\\nExome Sequencing\\nFull Transcriptome Sequencing\\nIndividual-Level Genomic Data\\nMixed\\nProbands\\nRepository\\nRNA Sequencing\\nSequencing\\nSingle Cell Analysis\\nTranscriptome Sequencing\\nTumorTumor vs. Matched-Normal\\nWhole Genome Sequencing\\nTotal number of consented subjects: 193  \n",
       "5                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            The goal for the University Texas PDX Development and Trial Center (UTPDTC) is to optimize personalized biomarker-based cancer therapy and identify effective targeted drugs based on the molecular characteristics of each tumor. Our short-term goals are to establish a biobank of clinically, and molecularly-annotated Patient-Derived Xenografts (PDXs) and to use PDXs as a platform for preclinical drug development and biomarker discovery. The primary goal for UTPDTC investigators will be to develop PDX trial strategies for preclinical testing of single agents and drug combinations. These models will allow the determination of the optimal treatments (single drugs or combinations) that should be tested in clinical trials in increasingly individualized, molecularly defined subsets of tumors. The goal of the Patient-Derived Xenograft Core is to provide high-quality clinically relevant and molecularly annotated PDX models for the research projects proposed in the University of Texas PDX Development and Trial Center (UTPDTC) grant application and to the research activity of the NCI PDX Development and Trial Centers Research Network (PDXNet) by leveraging PDX resources at our institutions and developing new PDX models from human cancer specimens using rigorous quality standards so that the models can be used to guide clinical trial development. The PDX models developed and/or characterized by the PDX Core will be available to other cancer researchers through the PDXNet and NCI Patient-Derived Models Repository (PDMR).\\n\\n    Study Design:\\n        Prospective Longitudinal Cohort\\n    Study Type:\\n        Cohort\\n    Total number of consented subjects: 36  \n",
       "6                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 The goal of this study is to deliver a detailed characterization of the patterns of disease dissemination at diagnosis, during progression and in response to therapy in high-risk neuroblastoma. Longitudinal and spatially distinct tumors were collected from patients. Clinical data includes diagnosis and treatment information. Biospecimen data includes whole genome sequencing (WGS) and whole transcriptome sequencing (WTS).\\n\\nStudy Design:\\nTumor vs. Matched-Normal\\nStudy Type:\\nLongitudinal Cohort\\nTranscriptome Sequencing\\nWhole Genome Sequencing\\nTotal number of consented subjects: 129  \n",
       "7                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     The primary objective of this specimen correlative study was two-fold: to provide a mechanism for the association of known molecular alterations with clinical outcomes, and to provide rapid genetic profiling of alterations with known clinical utility using tumor and germline specimens to support treatment decisions.  \n",
       "8                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  This data set is a collection of patient specimens from common and rare cancers that have been developed into patient-derived xenograft (PDX) models at the Washington University PDX Development and Trial Center (WU-PDTC). The goal of the PDTC is to develop and characterize PDX models, gaining insight into tumor biology, and validating biomarkers across all major tumor types. Pre-clinical experiments in these PDX models will be used to advance our ability to predict clinical responses to new molecularly targeted agents under development.\\n\\nStudy Design:\\nTumor vs. Matched-Normal\\nStudy Type:\\nTumor\\nTumor vs. Matched-Normal\\nTotal number of consented subjects: 127  \n",
       "9                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Recently, significant progress has been made in characterizing and sequencing the genomic alterations in statistically robust numbers of samples from several types of cancer. For example, The Cancer Genome Atlas (TCGA), International Cancer Genome Consortium (ICGC) and other similar efforts are identifying genomic alterations associated with specific cancers (e.g., copy number aberrations, rearrangements, point mutations, epigenomic changes, etc.) The availability of these multi-dimensional data to the scientific community sets the stage for the development of new molecularly targeted cancer interventions. Understanding the comprehensive functional changes in cancer proteomes arising from genomic alterations and other factors is the next logical step in the development of high-value candidate protein biomarkers. Hence, proteomics can greatly advance the understanding of molecular mechanisms of disease pathology via the analysis of changes in protein expression, their modifications and variations, as well as protein=protein interaction, signaling pathways and networks responsible for cellular functions such as apoptosis and oncogenesis. Realizing this great potential, the NCI launched the third phase of the CPTC initiative in September 2016. As the Clinical Proteomic Tumor Analysis Consortium, CPTAC continues to define cancer proteomes on genomically-characterized biospecimens. The purpose of this integrative approach was to provide the broad scientific community with knowledge that links genotype to proteotype and ultimately phenotype. In this third phase of CPTAC, the program aims to expand on CPTAC II and genomically and proteomically characterize over 2000 samples from 10 cancer types (Lung Adenocarcinoma, Pancreatic Ductal Adenocarcinoma, Glioblastoma Multiforme, Acute Myeloid Leukemia, Clear cell renal Carcinoma, Head and Neck Squamous Cell Carcinoma, Cutaneous Melanoma, Sarcoma, Lung Squamous Cell Carcinoma, Uterine Corpus Endometrial Carcinoma) .Germline DNA is obtained from blood and Normal control samples for proteomics varied by organ site. All cancer samples were derived from primary and untreated tumor.  \n",
       "10                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     The implementation of targeted therapies for acute myeloid leukaemia (AML) has been challenging because of the complex mutational patterns within and across patients as well as a dearth of pharmacologic agents for most mutational events. This supplement data from the Beat AML program provides Adolescent and Young Adult (AYA) and pediatric tumor specimens not previously released. These data include exome and RNA level characterization as well as analyses of ex vivo drug sensitivity. Collectively, this data can be leveraged to address clinical, genomic, transcriptomic and functional analyses of the biology of leukemia in pediatric and AYA populations.\\nStudy Design:\\nCase Set\\nStudy Type:\\nCase Set\\nExome Sequencing\\nRNA Sequencing\\nTotal number of consented subjects: 104  \n",
       "11                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  In this study, we will apply a multi-staged approach to reveal genes harboring rare variants that are associated with aggressive PCa. Whole-exome sequencing (Aim 1a) of 2,774 aggressive cases and 2,776 non-aggressive cases of European ancestry will be conducted followed by rare variant analysis of single sites and gene burden testing to identify novel susceptibility loci/genes for aggressive disease. We will validate the most significantly associated genes (~500) through targeted sequencing in an additional 7,500 aggressive and 7,500 non-aggressive cases (Aim 1b). Next, we will investigate the clinical predictive utility of the genes/variants identified in 2,300 cases in the STHM3 trial who are undergoing biopsy based on PSA and genetic risk score stratification (Aim 2). Through this tiered approach we expect to significantly advance knowledge of aggressive PCa etiology and health disparities as well as guide the development of early detection and prognostic strategies for the subset of men who are most susceptible to this fatal form of disease. In this case-case study of aggressive vs non aggressive prostate cancer, aggressive cases are defined as prostate cancer as cause of death, (T4 disease or T3 disease) and Gleason 8+. Non-aggressive cases are men with T1/2 disease and Gleason <=6.\\n\\nCONTRIBUTING SITES\\n\\nCAPS, PROCAP, STHM1, STHM2: Swedish Cancer Society (CAN 2016/818), Swedish Research Council (2014/2269)\\n\\nMEC: Funding provided by the National Cancer Institute: Understanding Ethnic Differences in Cancer, 2U01CA164973 and The Genetic Basis of Aggressive Prostate Cancer, The Role of Rare Variation, 5R01CA196931-02.\\n\\nCPSII: The authors express sincere appreciation to all Cancer Prevention Study II participants and to each member of the study and biospecimen management group. The American Cancer Society funds the creation, maintenance, and updating of the Cancer Prevention Study-II cohort.\\n\\nMCCS/APCS/PCFS: The Melbourne Collaborative Cohort Study (MCCS) recruitment was funded by VicHealth and Cancer Council Victoria and further supported by Australian National Health and Medical Research Council (NHMRC) grants 209057 and 396414. The Aggressive Prostate Cancer Case-Control Study (APCS) was funded by NHMRC grant 623204. The Prostate Cancer Family Study (PCFS) was fully funded by Cancer Council Victoria. Cancer Council Victoria funds the continuing maintenance and updating of the MCCS, APCS and PCFS. Cases and their vital status are ascertained and followed up through the Victorian Cancer Registry and the Australian Institute of Health and Welfare, including the National Death Index and the Australian Cancer Database.\\n\\nPLCO: The Prostate Lung Colorectal Ovarian Cancer Screening Trial (PLCO) was supported by the Intramural Research Program of the Division of Cancer Epidemiology and Genetics and by contracts from the Division of Cancer Prevention, National Cancer Institute, US National Institutes of Health, Department of Health and Human Services.\\n\\nATBC: The ATBC Study is supported by the Intramural Research Program of the U.S. National Cancer Institute, National Institutes of Health, and by U.S. Public Health Service contract HHSN261201500005C from the National Cancer Institute, Department of Health and Human Services.\\n\\nICR: This work was supported by the NIH R01 grant 5R01CA196931-02.The samples from the UK were from UKGPCS and PrompT. The UKGPCS study was supported by Cancer Research UK (grant numbers C5047/A7357, C1287/A10118, C1287/A5260, C5047/A3354, C5047/A10692, C16913/A6135 and C16913/A6835). We would like to acknowledge the NCRN nurses and Consultants for their work in the UKGPCS study. We thank all the patients who took part in this study. We also acknowledge The Institute of Cancer Research, The National Cancer Research Network UK, The National Cancer Research Institute (NCRI) UK for their ongoing support. We are grateful for support of NIHR funding to the NIHR Biomedical Research Centre at The Institute of Cancer Research and The Royal Marsden NHS Foundation Trust.\\n\\nCIDR grant X01HG008336\\n\\nStudy Design:\\nCase Set\\nStudy Type:\\nCase Set\\ndbGaP estimated ancestry using GRAF-pop\\nTotal number of consented subjects: 5563  \n",
       "12                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Comprehensive Whole Genome Sequencing and Whole Transcriptome (RNASeq) analyses from 114 pediatric patients at Memorial Sloan Kettering. This dataset was used as a prospective clinical utility and feasibility study. By using our Isabl infrastructure for precision medicine, we developed a 2-week end-to-end pipeline to analyze cfDNA, WGS and WTS, which we refer to as cWGTS  \n",
       "13  \"In RESPOND Project 2, we seek to identify rare genetic factors that are associated with prostate cancer (PCa) risk and aggressiveness in men of African ancestry (AA). We will conduct exome sequencing of 15,000 prostate cancer cases and 5,000 controls from the RESPOND cohort and the African Ancestry Prostate Cancer Consortium (AAPC) with cases selected based on risk categories: high-risk (stage T3/T4 or Gleason 8+ or PSA>20 ng/ml), intermediate-risk (stage T2b/T2c or Gleason 7 or PSA 10-20 ng/ml) and low-risk disease (stage T1/T2a and Gleason ≤ 6 and PSA<10 ng/ml). We expect the findings from this Project to significantly advance knowledge of susceptibility to aggressive PCa and racial/ethnic disparities in PCa risk, and to guide the development of future preventive, early detection and prognostic measures for AA men. The first phase of the study will include exome sequence data for approximately 7,500 cases and 5,000 controls from AAPC. The second phase of the study will include exome data for 5,000 cases from RESPOND and AAPC.\\n\\nACKNOWLEDGMENTS and CONTRIBUTING SITES\\n\\nMultiethnic Cohort (MEC): The MEC and the genotyping in this study were supported by National Institutes of Health (NIH) grants CA63464, CA54281, CA1326792, CA148085, and HG004726. Cancer incidence data for the MEC and Los Angeles Study of Aggressive Prostate Cancer (LAAPC) studies have been collected by the Los Angeles Cancer Surveillance Program of the University of Southern California with federal funds from the National Cancer Institute (NCI)/NIH/Department of Health and Human Services (DHHS) under Contract No. N01-PC-35139, and the California Department of Health Services as part of the state-wide cancer reporting program mandated by California Health and Safety Code Section 103885, and grant 1U58DP000807-3 from the Centers for Disease Control and Prevention (CDC).\\n\\nGhana Prostate Study (GPS): The Ghana Prostate Study was funded by the Intramural Program of the NCI/NIH/DHHS under Contract No. HHSN261200800001E.\\n\\nMen of African Descent and Carcinoma of the Prostate (MADCaP): We thank all MADCaP study participants. This work is a product of the MADCaP network. This work was supported by NCI/NIH grant U01CA184374 to Timothy Rebbeck and National Institute of General Medical Sciences (NIGMS) MIRA grant R35GM133727 to Joseph Lachance. Additional funding includes a seed grant from the Integrated Cancer Research Center at Georgia Institute of Technology.\\n\\nUGANDA: The UGANDA study was supported by NIH grant R01CA165862.\\n\\nSouthern Community Cohort (SCCS) is funded by NIH grant CA092447. SCCS sample preparation was conducted at the Epidemiology Biospecimen Core Lab that is supported in part by the Vanderbilt Ingram Cancer Center (CA68485). Data on SCCS cancer cases were provided by Alabama Statewide Cancer Registry, Kentucky Cancer Registry, Office of Cancer Surveillance at Tennessee Department of Health, Florida Cancer Data System, Central Cancer Registry at North Carolina Division of Public Health, Georgia Comprehensive Cancer Registry, Louisiana Tumor Registry, Mississippi Cancer Registry, South Carolina Central Cancer Registry, Virginia Cancer Registry at Virginia Department of Health, and Cancer Registry at Arkansas Department of Health. The Arkansas Central Cancer Registry is fully funded by a grant from National Program of Cancer Registries (NPCR)/Centers for Disease Control and Prevention (CDC). Data on SCCS cancer cases from Mississippi were collected by the Mississippi Cancer Registry which participates in the NPCR/CDC. The contents of this publication are solely the responsibility of the authors and do not necessarily represent the official views of the CDC or the Mississippi Cancer Registry.\\n\\nKaruprostate: The Karuprostate study was supported by the French National Health Directorate and by the Association pour la Recherche sur les Tumeurs de la Prostate. Séverine Ferdinand, Marc Romana.\\n\\nThe North Carolina - Louisiana Prostate Cancer Project (PCaP) is carried out as a collaborative study supported by the Department of Defense contract DAMD 17-03-2-0052. The authors thank the staff, advisory committees and research subjects participating in the PCaP study for their important contributions. We would also like to acknowledge the UNC BioSpecimen Facility and the LSUHSC Pathology Lab for our DNA extractions, blood processing, storage and sample disbursement (https://genome.unc.edu/bsp).\\n\\nGene-Environment Interaction in Prostate Study (GECAP) was supported by NIH grant ES011126.\\n\\nKing County Prostate Cancer Study (KCPCS) was supported by NIH grants CA056678, CA082664, and CA092579, with additional support from the Fred Hutchinson Cancer Research Center. We thank the participants in these studies, and Ms. Suzanne Kolb for help with study management.\\n\\nThe Los Angeles Study of Aggressive Prostate Cancer (LAAPC) was funded by grant 99-00524V-10258 from the Cancer Research Fund, under Interagency Agreement #97-12013 (University of California contract #98-00924V) with the Department of Health Services Cancer Research Program. Cancer incidence data for the MEC and LAAPC studies have been collected by the Los Angeles Cancer Surveillance Program of the University of Southern California with Federal funds from the NCI/NIH/DHHS under Contract No. N01-PC-35139, and the California Department of Health Services as part of the state-wide cancer reporting program mandated by California Health and Safety Code Section 103885, and grant 1U58DP000807-3 from the CDC.\\n\\nProstate Cancer Studies at MD Anderson (MDA) was supported by grants CA68578, ES007784, DAMD W81XWH-07-1-0645, and CA140388.\\nStudy Design:\\nCase-Control\\nStudy Type:\\nCase-Control\\nCohort\\nExome Sequencing\\nTotal number of consented subjects: 12049\"  \n",
       "14                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        This sequencing cohort defines the largest, clinically annotated pediatric brain tumor cohort study to date and seeks to define the intersection of germline and somatic underpinnings of pediatric brain tumors across a shared developmental context of cancer and structural birth defects.\\nStudy Weblinks:\\nChildrens Brain Tumor Network\\nPacific Pediatric Neuro-Oncology Consortium\\nCHOP Division of Genomic Diagnostics\\nKids First Data Resource Portal\\nPedCBio Portal\\nStudy Design:\\nTumor vs. Matched-Normal\\nStudy Type:\\nClinical Cohort\\nClinical Genetic Testing\\nClinical Trial\\nCopy Number Variation (CNV)\\nExome Sequencing\\nGenotype/Expression\\nIndividual-Level Genomic Data\\nLongitudinal\\nLongitudinal Cohort\\nMulticenter\\nObservational\\nParent-Offspring\\nParent-Offspring Trios\\nProbands\\nProspective\\nRNA Sequencing\\nSequencing\\nTissue Expression\\nTumor\\nTumor vs. Matched-Normal\\nWhole Genome Sequencing\\nTotal number of consented subjects: 1031  \n",
       "15                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     Overall the outcomes of patients with metastatic melanoma have improved dramatically over the last decade due to an improved understanding of the molecular drivers of this disease. In particular, multiple targeted therapy regimens have been approved for patients with a BRAFV600E/K mutation, which are present in ~50% of cutaneous melanomas. These treatments achieve clinical responses in ~80% of patients with a BRAFV600E/K mutation, thus providing proof-of-concept of the therapeutic potential for personalized therapeutic strategies. However, most of the patients will progress within 2 years of starting those therapies. Further, currently there are no targeted therapies that have been shown to be effective in patients with a wild-type BRAF. Thus, there are unmet clinical needs to develop treatments that prevent or overcome resistance to existing therapies for patients with a BRAFV600E/K mutation, and that are effective in patients without a BRAF mutation. In order to facilitate the development of new therapeutic strategies, over the last 5 years we have led a major effort to develop a broad collection of PDX models to reflect the clinical, histological, and genetic heterogeneity of this disease. Our collection of PDX models represents one of the largest collections for any human malignancy, and our initial testing demonstrates that the collection accurately recapitulates the oncogenic drivers and molecular heterogeneity that is observed in patients. This collection also includes a subset of PDX established from patients with acquired resistance to targeted therapies that have been maintained on those agents in vivo to sustain their resistant phenotype. Together these efforts have generated a robust resource to develop, refine, and prioritize new personalized combinatorial therapies for patients. Thus, we propose to establish a multi-disciplinary and multi-institutional PDTC Program focused on the use and continued expansion of our robust melanoma PDX collection to identify new therapeutic approaches that fill important clinical gaps in this disease.\\nStudy Design:\\nCase Set\\nStudy Type:\\nCase Set\\nExome Sequencing\\nRNA Sequencing\\nTotal number of consented subjects: 46  \n",
       "16                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  Even though whole genome sequence (WGS) data has been generated and published in many studies, much of this information is not yet processed for use in down-stream analysis. This study's aim is to provide 1342 WGS normal-tumor paired single nucleotide variations (SNP) over 18 different cancer types provided by The Cancer Genome Atlas (TCGA) project. Individual level data for TCGA can be accessed by requesting access for phs000178. This data includes variations within self-reported white and African-American populations. Variations that exist within the tumor tissue but are absent in associated normal organ tissue (as compared to the human reference genome) are reported. Data published includes SNP and small insertions and deletions, which were generated through a pipeline including the VarScan2 variant calling software.  "
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.set_option('display.max_colwidth', None)\n",
    "study_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "acfc0835-3add-4ecd-9ba5-d9a8da2ebdbd",
   "metadata": {},
   "source": [
    "From this list, HTAN looks interesting, let's take a look at some numbers to see if there's enough data there to support further analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "94ed3438-1541-40b1-b473-e1f78c6a5d0a",
   "metadata": {},
   "outputs": [],
   "source": [
    "info_query= \"\"\"\n",
    "{\n",
    "  studyDetail(phs_accession:\"phs001287\"){\n",
    "    data_types\n",
    "    numberOfFiles\n",
    "    numberOfSubjects\n",
    "    numberOfSamples\n",
    "  }\n",
    "}\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "a8e838cb-35c9-4a99-9268-c0077abb2769",
   "metadata": {},
   "outputs": [],
   "source": [
    "info_results = runGraphQLQuery(cds_graphql_url, info_query)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "7cbd941c-6f80-44aa-b579-673557388901",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'data': {'studyDetail': {'data_types': 'Genomic', 'numberOfFiles': 1243, 'numberOfSubjects': 1069, 'numberOfSamples': 1114}}}\n"
     ]
    }
   ],
   "source": [
    "print(info_results)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e57beca5-ba4d-48b0-8213-9f5e4ad12d1e",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
